genFigures.py

```diff
@@ -22,42 +22,42 @@
 
     print(f'Figure {sys.argv[1]} is done after {time.time() - start} seconds.\n')
 
-    if sys.argv[1] == '2':
+    if sys.argv[1] == 'S02':
         # Overlay Figure 2 cartoon
-        overlayCartoon(fdir + 'figure2.svg',
-                       f'{cartoon_dir}/figure2.svg', 0, 0, scalee=0.2)
+        overlayCartoon(fdir + 'figureS02.svg',
+                       f'{cartoon_dir}/figureS02.svg', 0, 0, scalee=0.2)
```

Comment:
@jclagarde Note the slight changes here. Once this goes on `master` and you merge with it, let me know if you're having any troubles.